src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -247,26 +247,30 @@ static void InputScrollRight(const ScreenCoordsXY& screenCoords, int32_t state)
 
     switch (state)
     {
-        case MOUSE_STATE_RELEASED:
+        case MouseState::Released:
             _ticksSinceDragStart += gCurrentDeltaTime;
             if (screenCoords.x != 0 || screenCoords.y != 0)
             {
                 _ticksSinceDragStart = 1000;
                 InputScrollDragContinue(screenCoords, w);
             }
             break;
-        case MOUSE_STATE_RIGHT_RELEASE:
+        case MouseState::RightRelease:
             _inputState = InputState::Reset;
             context_show_cursor();
             break;
+        default:
+        {
+            // Do nothing. Workaround for compilation errors.
```

Comment:
Make these comments more specific. No other states are relevent for a right mouse scroll interaction.